scripts/setup-test-data/src/setup_test_data/cli.py

```
@@ -287,142 +373,50 @@ def main(args: Optional[list[str]] = None) -> None:
-        OUTPUT_DIR = parsed.output_dir
-    if parsed.resources_dir is not None:
-        RESOURCES_DIR = parsed.resources_dir
-
     headers = get_headers()
-    with httpx.Client(
-        headers=headers, timeout=60.0, follow_redirects=True
-    ) as client:
-        for variant in parsed.variants:
-            repo = REPOS[variant]
-            print(f"\nProcessing {variant} ({repo})...")
+    with httpx.Client(headers=headers, timeout=None, follow_redirects=True) as client:
```

Ruff (0.15.15)

[error] Probable use of `httpx` call with timeout set to `None`

(S113)